.github/workflows/test.yml

```
@@ -0,0 +1,47 @@
+                  p/security-audit
+                  p/owasp-top-ten
+                  p/javascript
+                  p/python
+              run: semgrep ci
+    # TODO: Fix Bandit Vulns
+    "Bandit":
+      runs-on: ubuntu-latest
+      container: python:3.11
+  
+      steps:
+        - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)